docs/admin/configuration.md

```diff
@@ -28,25 +28,21 @@ bpfd configuration file (`/etc/bpfd/bpfd.toml`) to retrieve the bpfd-client cert
 bpfd allows the user to specify certain bpf programs to always be loaded every time the daemon is started.
 To do so simply create `.toml` files in the `/etc/bpfd/programs.d` directory with the following syntax:
 
-**Users can specify multiple programs in a single `.toml` file AND multiple `.toml` files** 
+**NOTE:** Users can specify multiple programs in a single `.toml` file AND multiple `.toml` files
 
 ```toml
 
 [[programs]]
 name = "program0"
-interface = "eth0"
-path = <PATH TO BPF BYTECODE>
-section_name = "pass"
+path = "/usr/local/src/xdp-tutorial/basic01-xdp-pass/xdp_pass_kern.o"
```

Comment:
Did this get left by accident, since couldn't the "PATH" here be different for every user?

Reply:
Ok as I read further I see you did this just for the sake of consistency with our main tutorial so I guess that's alright. :)

Reply:
Yes, it could be different per user, but was trying to get `/home/bmcfall` and `/home/dave` out of the examples.